renovate/src/Renovate/BinaryFormat/ELF.hs

```diff
@@ -628,7 +628,6 @@ choosePHDRSegmentAddress elf = do
   -- one can't be found.
   (tlssegment, segmentInfos) <- indexLoadableSegments elf fakePhdrSeg
   case findSpaceForPHDRs segmentInfos projectedOffset requiredSize of
-    NoAddress -> P.throwM $ NoSpaceForPHDRs (fromIntegral projectedOffset) (fromIntegral requiredSize)
```

Comment:
With this removed, can the `NoSpaceForPHDRs` error still be thrown? If not, we should probably remove it.

Reply:
It can still be thrown (see line 643, about 10 down from here).